src/layouts/SiteLaunchPad/components/SiteLaunchFinalState.tsx

```diff
@@ -0,0 +1,82 @@
+import { HStack, Text, VStack } from "@chakra-ui/react"
+import { Button } from "@opengovsg/design-system-react"
+
+import { useSiteLaunchContext } from "contexts/SiteLaunchContext"
+
+import { SiteLaunchFailureImage, SiteLaunchSuccessImage } from "assets"
+
+const SiteLaunchSuccessState = (): JSX.Element => {
+  return (
+    <>
+      <Text
+        as="h3"
+        textStyle="h3"
+        textColor="base.content.strong"
+        mb="1.5rem"
+        mt="3rem"
+      >
+        We have lift off! Your site has successfully launched.
+      </Text>
+      <SiteLaunchSuccessImage />
+
+      <Text textStyle="body-1" textColor="base.content.default" mt="1.5rem">
+        View your site status from the Dashboard.
+      </Text>
+      <Text textStyle="body-1" textColor="base.content.default" mb="3rem">
+        You can find your site&apos;s DNS records in the settings page.
+      </Text>
+      <HStack spacing="3rem">
+        <Button variant="link">Back to tasklist</Button>
+        <Button>Visit live site</Button>
+      </HStack>
+    </>
+  )
+}
+
+const SiteLaunchFailureState = (): JSX.Element => {
+  return (
+    <>
+      <Text
+        as="h3"
+        textStyle="h3"
+        textColor="base.content.strong"
+        mb="1.5rem"
+        mt="3rem"
+      >
+        Site failed to launch, please contact the Isomer team
+      </Text>
+      <SiteLaunchFailureImage />
+
+      <Text
+        textStyle="body-1"
+        textColor="base.content.default"
+        mt="1.5rem"
+        textAlign="center"
+      >
+        The most common reason for site launch failure is unknown cloudflare
+        records. Please remove all existing domains and try site launch again.
```

Comment:
nit: are we standardizing to `Cloudflare` haha

Reply:
omg is actually cloudfront this slipped my brain

Reply:
thanks for the catch 7525244b16b989c3b82b68001e4d9914327cef44